pages/developers/style.md

```diff
@@ -219,17 +219,33 @@ files = src
 python_version = 3.6
 warn_unused_configs = True
 
+# Currently (0.800) identical to --strict
+disallow_any_generics = True
+disallow_subclassing_any = True
+disallow_untyped_calls = True
+disallow_untyped_defs = True
+disallow_incomplete_defs = True
+check_untyped_defs = True
+disallow_untyped_decorators = True
+no_implicit_optional = True
+warn_redundant_casts = True
+warn_unused_ignores = True
+warn_return_any = True
+no_implicit_reexport = True
+strict_equality = True
+
 [mypy-numpy.*]
 ignore_missing_imports = True
 ```
 
 There are a lot of options, and you can start with only typing global code and
 functions with at least one type annotation (the default) and enable more
-checks as you go. You can ignore missing imports on libraries as shown above,
-on section each. And you can disable MyPy on a line with `# type: ignore`. Once
-you are ready to start checking more, you can look at adding
-`check_untyped_defs`, `disallow_untyped_defs`, `disallow_incomplete_defs`, and
-more. You can add these *per file* by adding a `# mypy: <option>` the top of
+checks as you go (possibly by slowly uncommenting items in the list above).
+You can ignore missing imports on libraries as shown above,
+on section each. And you can disable MyPy on a line with `# type: ignore`.
```

Comment:
What do you mean by "on section each"? Words in the wrong order I guess.

Reply:
one section each.